### PR TITLE
Fix #609 — Handle boilerplate db_type conf in sympa.pl --health_check

### LINKED ISSUE #609: Improving handling of boilerplate configuration
```
# /sbin/sympa.pl --health_check
err Conf::_set_listmasters_entry() Robot (You must define this parameter) config: Listmaster address "(Youmustdefinethisparameter)" is not a valid email
err Conf::_set_listmasters_entry() Robot (You must define this parameter) config: All the listmasters addresses found were not valid. Out of 1 addresses provided, 0 only are valid email addresses
Bareword found where operator expected at (eval 279) line 1, near "require Sympa::DatabaseDriver::(You"
	(Do you need to predeclare require?)
Database sympa defined in sympa.conf has not the right structure or is unreachable. verify db_xxx parameters in sympa.conf
```
The command above is designed to test Sympa configuration and thus shouldn't crash on the boilerplate configuration file. It should certainly not try to load a database driver in case the `db_type` is obviously invalid.

Version
-------
6.2.42

Installation method
-------------------
RPM

### PR DESCRIPTION
**Pull request by ldidry**
Fix #609

### CODE REVIEW

**Comment by racke** [comment]
Good ... a bit rewording maybe?

> is the boilerplate one and is obviously incorrect

I suggest to drop the second "is", otherwise it looks like you are talking about unrelated things.

**Comment by ldidry** [comment]
Ok. Reworded :slightly_smiling_face:

File: src/sbin/sympa.pl.in [hunk 1]
```diff
@@ -212,10 +212,17 @@ if ($main::options{'upgrade_config_location'}) {
             $Conf::Conf{'etc'};
     }
 
+    ## Check if db_type is not the boilerplate one
+    if ($Conf::Conf{'db_type'} eq '(You must define this parameter)') {
+        die sprintf
```
Comment (ikedas) [inline src/sbin/sympa.pl.in]: I feel It'd better to be like
```
    ...
    unless ($Conf::Conf{'db_type'} and $Conf::Conf{'db_type'} =~ /\A\w+\z/) {
        die sprintf
    ...
```

Comment (ldidry) [reply src/sbin/sympa.pl.in]: I added this check in 0807b7972e4d9506fbc70bdddb7723cbefcd5a63, but left the first one: this check does not allow us to be sure it’s the boilerplate `db_type`, so the `die` message has to be different.
Is it ok for you?

Comment (ikedas) [reply src/sbin/sympa.pl.in]: I'm ok for your change. Thanks!

**Pull request merged**